lib/commands.py

```diff
@@ -657,10 +657,23 @@ def is_synchronized(self):
         return self.wallet.is_up_to_date()
 
     @command('n')
-    def getfeerate(self):
-        """Return current optimal fee rate per kilobyte, according
-        to config settings (static/dynamic)"""
-        return self.config.fee_per_kb()
+    def getfeerate(self, fee_method=None, fee_level=None):
```

Comment:
Does it make sense to have `fee_level` default to `None`. Doing so complicates the code with several if statements. Also passing a default signals the argument is optional which it does seem to be.

Reply:
both arguments are optional. if they are not passed, we deduce their value from the config.
this also makes this change backwards compatible